model_analyzer/config/generate/automatic_model_config_generator.py

```diff
@@ -96,16 +96,10 @@ def _done_walking_instance_count(self):
     def _max_batch_size_limit_reached(self):
         return (self._curr_max_batch_size * 2) > self._max_model_batch_size
 
-    def _last_results_erroneous(self):
```

Comment:
Why is this being removed? Is it just dead code?

Reply:
Yes. Deadcode. I had started to use it in the solution but ran into a failure in a different unit test that showed it was the wrong way to go.